### ISSUE TITLE
Accessibility issue: tab doesn't work in authentication prompt for ICP4D

### ISSUE THREAD

**Issue by natashadsilva** [IBMStreams/build-ibmstreams#55]
Tab doesn't seem to work when trying to log in to the ICP4D instance. Only affects Atom
